src/utils/chartJs/redashFormatter.js

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable camelcase */
+
 import generateDatasetStyle from './generateDatasetStyle';
 import SETTINGS from '../../settings';
 
 const sortByDate = (a, b) =>
   new Date(b.submission_date) - new Date(a.submission_date);
 const dataToChartJSformat = data =>
-  // eslint-disable-next-line camelcase
```

Comment:
We only disable specific lines rather than the whole file so we don't regress other parts of the file.